crates/cxx-qt-gen/src/generator/cpp/property/mod.rs

```diff
@@ -186,15 +177,14 @@ pub mod tests {
         )
         .unwrap();
 
-        // should be a pair
-        let result = require_header(&generated.methods[0]);
-        assert!(result.is_err());
-
-        let result = require_pair(&generated.private_methods[0]);
+        // should be a header
+        let result = require_pair(&generated.methods[0]);
         assert!(result.is_err());
 
         let result = require_source(&generated.methods[0]);
         assert!(result.is_err());
```

Comment:
the above checks for a pair so these lines can be removed?

Reply:
Like the call above I think this just exists to get 100% coverage on require_source for the error case.